index.js

```diff
@@ -1,84 +1,82 @@
+// eslint-disable-next-line max-classes-per-file
 const generateId = () => (Math.random() + 1).toString(36).substring(7);
```

Comment:
- [ ] **As the previous reviewer mentioned, any operation related to the creation of a book should be handled by the class itself. Please fix it.**

In the object-oriented paradigm, the classes help us to encapsulate the logic for any object. This means that any property or action related to any object will be contained in its class. This helps us to implement interfaces and practice separation. Please move the logic to generate a book id within the book class. You can define it as a method of the class.